exercises/C-comparison-operators/exercise.js

```diff
@@ -7,15 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity >= (studentCount +mentorCount)// finish this statement
```

Comment:
Great job. You can put the sum of studentCount and mentorCount in a variable and then compare roomMaxCapacity with that variable